cmd/setup.go

```diff
@@ -321,3 +319,37 @@ func configureLoadpoints(conf config, cp *ConfigProvider) (loadpoints []*core.Lo
 
 	return loadpoints, nil
 }
+
+// configureCircuits loads circuit definition and connects the loadpoints with the circuit
+func configureCircuits(site *core.Site, loadPoints []*core.Loadpoint, cp *ConfigProvider) (err error) {
+	for _, cfg := range conf.Circuits {
+		circuit, err := core.NewCircuitFromConfig(cp, cfg, site)
+		if err != nil {
+			return fmt.Errorf("failed configuring circuit: %w", err)
+		}
+		site.Circuits = append(site.Circuits, circuit)
+	}
+
+	// connect circuits and loadpoints
+	for _, lpId := range loadPoints {
```

Comment:
Es wäre schön, wenn wir das dem Ladepunkt überlassen könnten so wie der sich auch die Zähler und Fahrzeuge raus sucht. Wäre das möglich? Inversion of control?

Reply:
Ist das noch ein Thema? Aktuell werden zuerst die LP angelegt, dann die Site mit den LP als Input. Circuits sind der Site bekannt.
Ich kann die Reihenfolge verändern und erst die Circuits anlegen, und dann die LP. Dann sollte ein LP seinen Circuit finden können.
Decision?